packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth.ts

```diff
@@ -2102,119 +2101,6 @@ describe("Eth module", function () {
           assert.notEqual(logs2[0].address, "changed");
         });
 
-        it("Should accept block hashes as from", async function () {
```

Comment:
Some tests were removed because this RPC methods don't accept the new kind of blocktag. I checked this with geth's code.